nipype/algorithms/compcor.py

```diff
@@ -22,6 +22,15 @@ class CompCoreOutputSpec(TraitedSpec):
 class CompCore(BaseInterface):
     '''
     Node with core CompCor computation, used in aCompCor and tCompCor
+
+    Example
+    -------
+
+    >>> ccinterface = CompCore()
```

Comment:
I don't see the file header I mention in my previous comment it your code. Did you try adding it? Also notice that the after adding the header the input files should be treated as if they were in the CWD: https://github.com/nipy/nipype/blob/master/nipype/interfaces/afni/preprocess.py#L45